openconnect_sso/config.py

```diff
@@ -62,6 +62,7 @@ class HostProfile(ConfigNode):
     address = attr.ib(converter=str)
     user_group = attr.ib(converter=str)
     name = attr.ib(converter=str)  # authgroup
+    on_disconnect = attr.ib(converter=str)
```

Comment:
If you'd like to persist this option, please put this attribute in the `Config` class instead. This is not relates to the connection details.

Reply:
Agreed. Will move.